cmd/jaeger-mongodb/main.go

```diff
@@ -18,15 +19,19 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.mongodb.org/mongo-driver/mongo/writeconcern"
 
-	"jaeger-mongodb/internal/jager-mongodb"
+	jager_mongodb "jaeger-mongodb/internal/jager-mongodb"
 )
 
 var configPath string
+var defaultConfigPath = "run/default-config.yaml"
 
 func main() {
 	flag.StringVar(&configPath, "config", "", "A path to the plugin's configuration file")
 	flag.Parse()
 
+	fmt.Println("Current config path:")
```

Comment:
you can do this on one line like `fmt.Println(fmt.Sprintf("Current config path: %s", configPath))`

https://golang.org/pkg/fmt/#Sprintf

Reply:
Awesome, thanks for the tips!